### DISCUSSION TITLE
Add support for notebook and PCMCIA emulation

### DISCUSSION THREAD

**Discussion by Fubuki0x10DE** [86Box/86Box#4045] [Ideas]
Add support for PCMCIA
Emulate a battery and always AC-plug
Difficulty is to compatible with manufacturer software, such as SONY / VAIO Power Manager

**Comment by upintheairsheep** [comment]
and don't forget the lid button, also specifications of the battery, AC adapter, and lid button can be found on https://lore.kernel.org/qemu-devel/20210120205501.33918-2-lb.workbox@gmail.com/T/#m0e416e3270a19850db9ca009bf13319f97505290
